internal/resources/aws/lambda_function.go

```diff
@@ -86,6 +91,16 @@ func (a *LambdaFunction) BuildResource() *schema.Resource {
 	},
 	)
 
+	if a.MonthlyRequests != nil {
+		monthlyRequests = decimalPtr(decimal.NewFromInt(*a.MonthlyRequests))
+		gbSeconds = decimalPtr(calculateGBSeconds(memorySize, averageRequestDuration, *monthlyRequests))
+		storageGBSeconds = decimalPtr(calculateStorageGBSeconds(storageSize, *gbSeconds))
+
+		costComponents = append(costComponents, a.storageCostComponent(storageGBSeconds, storageType))
+	} else {
+		costComponents = append(costComponents, a.storageCostComponent(decimalPtr(decimal.NewFromInt(0)), storageType))
```

Comment:
```suggestion
		costComponents = append(costComponents, a.storageCostComponent(nil, storageType))
```
I think this means it will show the pricing instead of a zero cost, which is what we want if there's no usage data added.